src/movement/thruster_sabertooth.cpp

```diff
@@ -117,5 +149,12 @@ int main(int argc, char **argv)
 
   ros::spin();
 
+  uint8_t serial_data[4];
```

Comment:
This code shouldn't be here (152-157). All of the work should be (and is) done in the callback function.